dietpi/func/dietpi-set_hardware

```diff
@@ -2259,6 +2259,96 @@ _EOF_
 
 	}
 
+	#/////////////////////////////////////////////////////////////////////////////////////
+	# Dedicated GPU
+	# INPUT_DEVICE_VALUE
+	# - Uninstall
+	# - Intel
+	# - Nvidia
+	# - AMD
+	#/////////////////////////////////////////////////////////////////////////////////////
+	DedicatedGPU_Main(){
+	
+		#Current GPU
+		local gpu_current=$(sed -n '/^[[:blank:]]*CONFIG_GPU_DRIVER=/{s/^[^=]*=//p;q}' /boot/dietpi.txt)
+		gpu_current=${gpu_current,,}
+		INPUT_DEVICE_VALUE=${INPUT_DEVICE_VALUE,,}
```

Comment:
Already lower-case, see above 🙂:
```suggestion
```

Reply:
> Already lower-case, see above slightly_smiling_face, but fallback is nice to have:

Yes. Covers user use of ```dietpi-set_hardware dedicatedgpu ITEM``` and future code.

Reply:
User use is forced lower-case as well. When I find time, I want to remove the `INPUT_DEVICE_VALUE=${INPUT_DEVICE_VALUE,,}` at the top of the script indeed and require explicit cases. In this case, also here `INPUT_DEVICE_VALUE=${INPUT_DEVICE_VALUE,,}` would be removed and instead we pass it in lower case to the script in the first place, and fail if cases are wrong. See my other comment to make them lower-care in `dietpi.txt` already.

Reply:
> You mean in dietpi-config? 

Yeah I did, my bad :smile: 

> But as you mentioned "DietPi-Software": We could actually prompt the menu /boot/dietpi/dietpi-config 2 during Steam install, as we do on next DietPi update. At least if the current selection is still "none".

Not a bad idea :+1: . I'd probably move it to X11 install. Prompt there if no driver set?